audbackend/core/api.py

```diff
@@ -85,7 +85,7 @@ def access(
 
     Examples:
         >>> access('file-system', 'host', 'doctest')
-        ('audbackend.core.filesystem.FileSystem', 'host', 'doctest')
+        ('audbackend.core.conftest.DoctestFileSystem', 'host', 'doctest')
```

Comment:
One way to handle this would be to overwrite `__repr__` inside `DoctestFileSyetem` as well, e.g.

```python
    def __repr__(self) -> str:
        module = f'{audbackend.FileSystem.__class__.__module__}'
        class = f'{audbackend.FileSystem.__class__.__name__}'
        return str((f'{module}.{class}', self.host, self.repository))
```

Reply:
Good idea, we now output `'audbackend.core.filesystem.FileSystem'` again.